src/dataIntegration/transcriptomics/FASTCORE/fastcore.m

```diff
@@ -1,119 +1,155 @@
-function A = fastcore(C, model, epsilon, printlevel)
-% The FASTCORE algorithm for context-specific metabolic network reconstruction.
-% Input C is the core set, and output A is the reconstruction
+function tissueModel = fastcore(core, model, epsilon, expressionRxns, threshold, printlevel)
+% Use the FASTCORE algorithm (`Vlassis et al, 2014`) to extract a context
+% specific model using data. FASTCORE algorithm defines one set of core
+% reactions that is guaranteed to be active in the extracted model and find
+% the minimum of reactions possible to support the core.
 %
 % USAGE:
 %
-%    A = fastcore( C, model, epsilon )
+%    tissueModel = fastcore(core, model, epsilon, expressionRxns, threshold, printlevel)
 %
 % INPUTS:
-%    C:             indices of reactions in cobra model that are part of the
-%                   core set of reactions
-%    model:         cobra model structure containing the fields
+%    core:              indices of reactions in cobra model that are part of the
+%                       core set of reactions (called `C` in `Vlassis et al,
+%                       2014`)
+%    model:             input model (COBRA model structure)
 %
-%                     * S - `m` x `n` stoichiometric matrix
-%                     * lb - `n` x 1 flux lower bound
-%                     * ub - `n` x 1 flux uppper bound
-%                     * rxns - `n` x 1 cell array of reaction abbreviations
-%    epsilon:       {1e-4} smallest flux that is considered nonzero
-%    printLevel:    0 = silent, 1 = summary, 2 = debug
+% OPTIONAL INPUTS:
+%    epsilon:           smallest flux value that is considered nonzero
+%                       (default 1e-8)
+%    expressionRxns:    expression data, corresponding to model.rxns (see
+%                       mapGeneToRxn.m)
+%    threshold:         expression threshold (reactions with expression
+%                       above this threshold are put in the set of core
+%                       reactions
+%    printLevel:        0 = silent, 1 = summary, 2 = debug
 %
 % OUTPUT:
-%    A:             indices of reactions in the new model
+%   tissueModel:         extracted model
+%
+%
+% `Vlassis, Pacheco, Sauter (2014). Fast reconstruction of compact
+% context-specific metbolic network models. PLoS Comput. Biol. 10, e1003424.`
 %
 % .. Authors:
 %       - Nikos Vlassis, Maria Pires Pacheco, Thomas Sauter, 2013 LCSB / LSRU, University of Luxembourg
 %       - Ronan Fleming, commenting of code and inputs/outputs
+%       - Anne Richelle, code adaptation to fit with createTissueSpecificModel
+
+
+    if ~exist('printLevel','var')
+        %For Compatability with the original fastcore syntax
+        printlevel = 1;
+    end
+
+    %Define the set of core reactions
```

Comment:
I am sure that with a generous collegial approach, yee can come to some compromise that